### PR TITLE
Anchor Platform: Update RequestOnchainFunds RPC doc

### PR DESCRIPTION
**Pull request by philipliu**
This updates the description of the `amount_out` field in the `RequestOnchainFunds` RPC documentation.

### CODE REVIEW

File: static/assets/rpc-methods/request_onchain_funds.json
Comment (JakeUrban) [inline static/assets/rpc-methods/request_onchain_funds.json:55]: - I think we want to remote "exchange" in this sentence, since transactions where `amout_in.asset` and `amount_out.asset` are the same are "exchange" transactions in the way we use the word in the protocol
- I think we need to change `asset_in` to `amount_in.asset` and `asset_out` for `amount_out.asset`

**Review by JakeUrban — APPROVED** [review]
LGTM after addressing my comment!

**Review by Ifropc — APPROVED** [review]

**Review by Ifropc — CHANGES_REQUESTED** [review]
Let's wait until [versioning pr is merged](https://github.com/stellar/stellar-docs/pull/909)

**Comment by Ifropc** [comment]
Actually I think you'd need to re-do this PR based off the versioned commit

**Pull request closed without merging**
